src/components/Popups/ReplyField/__tests__/ReplyFieldContainer-test.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import { Provider } from 'react-redux';
+import ReplyField from '../ReplyField';
+import ReplyFieldContainer, { Props } from '../ReplyFieldContainer';
+import { createStore } from '../../../../store';
+
+jest.mock('../ReplyField');
+
+describe('components/Popups/ReplyFieldContainer', () => {
+    const defaults = {
+        onChange: jest.fn(),
+        onClick: jest.fn(),
+    };
+    const store = createStore({
+        creator: {
+            cursor: 0,
+        },
+    });
+
+    const getWrapper = (props = {}): ReactWrapper<Props> =>
+        mount(
+            <Provider store={store}>
```

Comment:
You should be able to pass `store` directly to the `ReplyFieldContainer`, no provider needed.